pages/caritonaranjo/index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>carousel</title>
+    <link href="https://fonts.googleapis.com/css2?family=Tenali+Ramakrishna&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+    <link rel="carolina naranjo" href="humans.txt" />
+    <script src="js/main.js"></script>
+  </head>
+  <body>
+    <!--Carousel Wrapper-->
+    <section id="carousel">
```

Comment:
BEM only allows classes, but here  an Id is being used.

Reply:
the ID,  is not use for the style